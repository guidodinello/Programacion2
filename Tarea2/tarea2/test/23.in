# Implementar cambiarTodos
cambiarTodos 5 3
imprimirCadena
insertarAlFinal (5,1.3)
cambiarTodos 4 3
imprimirCadena
cambiarTodos 5 3
imprimirCadena
insertarAlFinal (6,1.2)
cambiarTodos 6 3
imprimirCadena
cambiarTodos 4 3
imprimirCadena
finalCadena
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (9,2.6)
cambiarTodos 5 5
imprimirCadena
cambiarTodos 9 5
imprimirCadena
cambiarTodos 3 5
imprimirCadena
cambiarTodos 5 8
imprimirCadena
Fin
