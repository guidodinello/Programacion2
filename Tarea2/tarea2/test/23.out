1>#  Implementar cambiarTodos.
2>Cambiados.
3>
4>Insertado al final.
5>Cambiados.
6>(5,1.30)
7>Cambiados.
8>(3,1.30)
9>Insertado al final.
10>Cambiados.
11>(3,1.30)(3,1.20)
12>Cambiados.
13>(3,1.30)(3,1.20)
14>loc al final.
15>Segmento insertado despuÃ©s de loc.
16>Cambiados.
17>(3,1.30)(3,1.20)(5,3.40)(8,9.40)(9,2.60)
18>Cambiados.
19>(3,1.30)(3,1.20)(5,3.40)(8,9.40)(5,2.60)
20>Cambiados.
21>(5,1.30)(5,1.20)(5,3.40)(8,9.40)(5,2.60)
22>Cambiados.
23>(8,1.30)(8,1.20)(8,3.40)(8,9.40)(8,2.60)
24>Fin.
